src/app/components/speakers/speakers.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="container justify-content-center">
+    <div class="row">
+        <div *ngFor="let speaker of (showAll ? (allSpeakers$ | async) : (allSpeakers$ | async | slice:0:3))"
```

Comment:
probably not the best way to do it as you have to async pipes which would be 2 subscriptions.

Better wrap it like this:

`<ng-container *ngIf="allSpeakers$ | async; let speakers">
  <div *ngFor="let speaker of (showAll ? speakers : (speakers | slice:0:3)"></div>
</ng-container>`

And please also include a loading template (can be a css loading indicator of your choice (just adjust colors accordingly and put it into its own component to make it reusable)) which is shown until allSpeakers$ is ready (api calls can take some seconds, at the moment its instantly there because its mocked data)